molecule/core.py

```diff
@@ -28,6 +28,8 @@
 from molecule import util
 from molecule.driver import basedriver
 
+from shutil import rmtree
```

Comment:
We do imports in a particular order.  Since this is a module bundled with python it should be imported under os on line `22`.  We also have a convention to import the base module, so code should be `import shutil` and reference `shutil.rmtree`.